### PR TITLE
solution

### PR DESCRIPTION
**Pull request by lefteris-L**
[DEMO LINK](https://lefteris-l.github.io/react_dynamic-list-of-goods/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,27 +1,62 @@
...
+    <div className="App">
+      <h1>Dynamic list of Goods</h1>
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+      <button
+        type="button"
+        data-cy="all-button"
+        onClick={() => loadGoods(getAll, Load.All)}
```
Comment (UmizDemud) [inline src/App.tsx:36]: The logic for the first parameter can be moved to inside of the loadGoods function

File: src/App.tsx [hunk 2]
```diff
@@ -1,27 +1,62 @@
...
+        data-cy="red-button"
+        onClick={() => loadGoods(getRedGoods, Load.Red)}
+      >
+        Load red goods
+      </button>
 
-    <GoodsList goods={[]} />
-  </div>
-);
+      {goods.length > 0 && (
```
Comment (UmizDemud) [inline src/App.tsx:57]: this can be 

```js
{goods.length && (

**Review by UmizDemud — APPROVED** [review]
I went through twice. Just two comments that aren't crucial. Approving, so that you can decide whether they make sense

File: src/App.tsx (hunk 2 above)
Comment (lefteris-L) [reply src/App.tsx:57]: It needs to be explicit boolean cause in some cases the number shows up next to the element

**Review by naziamoff — APPROVED** [review]
nice job 👍

**Pull request closed without merging**